backends/tc/introspection.h

```diff
@@ -97,6 +97,7 @@ struct ActionAttributes {
 
 struct TableAttributes {
     cstring name;
+    cstring permissions;
```

Comment:
Please add new field in the constructor as well

Reply:
addressed